tests/static_data/single_publication/layers_test.py

```diff
@@ -321,3 +321,15 @@ def test_layer_attributes_in_db(workspace, publ_type, publication):
     with app.app_context():
         attr_names = {col.name for col in layer_db.get_all_column_infos(workspace, publication)}
     assert attr_names == expected_names
+
+
+@pytest.mark.parametrize('workspace, publ_type, publication', [
+    publ_tuple for publ_tuple in data.LIST_LAYERS
+    if data.PUBLICATIONS[publ_tuple][data.TEST_DATA].get('external_images') is not None
+])
+@pytest.mark.usefixtures('liferay_mock', 'ensure_layman')
+def test_layer_style_external_images(workspace, publ_type, publication):
+    ensure_publication(workspace, publ_type, publication)
+
```

Comment:
Test that set of filenames in `external_images` directory is the same as 'external_images' set